server/views/pages/search/searchPrisoners.njk

```diff
@@ -9,7 +9,8 @@
   <div class="govuk-grid-row govuk-!-margin-bottom-4" data-qa="prisoner-search-section">
     <div class="govuk-grid-column-full">
       <div id="prisoner-search-container">
-        <h1 class="govuk-heading-xl">Search for prisoners</h1>
+        <h1 class="govuk-heading-xl">Search for a prisoner</h1>
+        <p>You can search using a person’s name or prison number.</p>
```

Comment:
Think this needs `govuk-body` else it will look strange with Times New Roman font